build.gradle

```
@@ -9,7 +9,12 @@ repositories {
 }
 
 dependencies {
-    compile('ch.qos.logback:logback-classic:1.2.3')
-    testCompile("org.junit.jupiter:junit-jupiter-api:5.6.0")
-    testCompile("org.assertj:assertj-core:3.11.1")
+    implementation 'org.junit.jupiter:junit-jupiter:5.4.2'
+    implementation 'junit:junit:4.12'
+    implementation 'junit:junit:3.8.2'
+    implementation 'org.junit.jupiter:junit-jupiter:5.4.2'
+    compile'ch.qos.logback:logback-classic:1.2.3'
+    testCompile "org.junit.jupiter:junit-jupiter-api:5.6.0"
+    testCompile "org.assertj:assertj-core:3.11.1"
+    testImplementation 'org.assertj:assertj-core:3.6.2'
```

```suggestion
    implementation 'ch.qos.logback:logback-classic:1.2.3'
    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.6.0'
    testImplementation 'org.assertj:assertj-core:3.11.1'
```

로 변경해보세요.
중복된게 너무 많습니다.

---

넵

---

@rla36
제가 조금 수정한 부분이 있는데, 이미 logback관련 코드가 있어서 추가했습니다.